docs/network.md

```diff
@@ -0,0 +1,55 @@
+# 10. Message transportation layers.
+
+This section describes different protocols of sending messages currently used in `Near`
+
+## 10.1 Messages between Actors.
+
+`Near` is build on `Actix`'s `actor` framework. (https://actix.rs/book/actix/sec-2-actor.html)
+Usually each actor runs on its own dedicated thread.
```

Comment:
do we have cases, where we have multiple 'copies' of the actors (therefore running on a few separate threads?)

Reply:
Yes. some of them have one thread per instance.